quacc/recipes/qchem/core.py

```diff
@@ -0,0 +1,627 @@
+    mult
+        The spin multiplicity of the molecular system.
+        Effectively defaults to the lowest spin state given the molecular structure and charge.
+    xc
+        Exchange-correlation functional.
+        Defaults to wB97M-V.
+    basis
+        Basis set.
+        Defaults to def2-SVPD.
+    scf_algorithm
+        Algorithm used to converge the SCF.
+        Defaults to DIIS, but for particularly difficult cases, GDM should be employed instead.
+    pcm_dielectric
+        Dielectric constant of the optional polarizable continuum impicit solvation model.
+        Defaults no None, in which case PCM will not be employed.
+    smd_solvent
+        Solvent to use for SMD implicit solvation model. Examples include "water", "ethanol", "methanol",
+        and "acetonitrile". Refer to the Q-Chem manual for a complete list of solvents available.
+        Defulats to None, in which case SMD will not be employed.
+    swaps
+        Dictionary of custom kwargs for the calculator. Must be formatted consistently with Pymatgen's
+        QChemDictSet's overwrite_inputs. For example: {"rem": {"symmetry": "true"}}.
+    opt_swaps
+        Dictionary of custom kwargs for run_ase_opt
+            opt_defaults = {"fmax": 0.01, "max_steps": 1000, "optimizer": "Sella"}
+
+    Returns
+    -------
+    dict
+        Dictionary of results from quacc.schemas.ase.summarize_opt_run
+    """
+
+    # Reminder to self: exposing TRICs?
+
+    true_charge, true_spin = true_charge_and_spin(atoms, charge, mult)
+
+    opt_swaps = opt_swaps or {}
+    opt_defaults = {
+        "fmax": 0.01,
+        "max_steps": 1000,
+        "optimizer": Sella,
+        "optimizer_kwargs": {},
+    }
+    opt_flags = opt_defaults | opt_swaps
+    if "sella.optimize" in opt_flags["optimizer"].__module__:
+        opt_flags["optimizer_kwargs"]["order"] = 0
+
+    if pcm_dielectric is not None and smd_solvent is not None:
+        raise ValueError("PCM and SMD cannot be employed simultaneously! Exiting...")
+
+    overwrite_inputs = {"rem": {"method": xc}}
+
+    swaps = swaps or {}
+    for key in swaps:
+        if key not in overwrite_inputs:
+            overwrite_inputs[key] = {}
+            for subkey in swaps[key]:
+                overwrite_inputs[key][subkey] = swaps[key][subkey]
+
+    qchem_input_params = {
+        "basis_set": basis,
+        "scf_algorithm": scf_algorithm,
+        "qchem_version": 6,
+        "pcm_dielectric": pcm_dielectric,
+        "smd_solvent": smd_solvent,
+        "overwrite_inputs": overwrite_inputs,
+    }
+
+    if scf_algorithm.lower() == "gdm":
+        qchem_input_params["max_scf_cycles"] = 200
+
+    calc = QChem(
+        input_atoms=atoms,
+        cores=cores,
+        charge=charge,
+        spin_multiplicity=mult,
+        qchem_input_params=qchem_input_params,
+    )
+    atoms.calc = calc
+    dyn = run_ase_opt(atoms, **opt_flags)
+
+    return summarize_opt_run(
+        dyn,
+        check_convergence=check_convergence,
```

Comment:
Perhaps it makes more sense for us to remove `check_convergence` as a kwarg from all the jobs and, in each summarize function, have `check_convergence: bool = SETTINGS.CHECK_CONVERGENCE` and link that to the global settings, independent of Q-Chem. Thoughts? As always, the user can override it temporarily with an environment variable or permanently by modifying the YAML.